lambdas/ingest/handler.py

```
@@ -0,0 +1,154 @@
+    """
+    records = event.get("Records", [])
+    logger.info("Processing %d SNS record(s)", len(records))
+
+    results = []
+    errors = []
+
+    for record in records:
+        try:
+            result = process_record(record)
+            results.append(result)
+        except (InvalidMessageError, Exception) as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)